toolhub/templates/home.jinja

```diff
@@ -1,5 +1,61 @@
 {% extends "layouts/base.jinja" %}
+{% from "partials/badges.jinja" import taxonomy_list, availability_badge %}
 
 {% block content %}
-    Home is where the heart is.
+    {% if user.is_authenticated %}
+        <div class="row">
+            <div class="col-sm-6 col-md-4 mb-4">
```

Comment:
I think it would be helpful to users if borrowed tools show up first when browsing on mobile devices. To keep the tools first visually on desktop we can use the ordering classes.

https://getbootstrap.com/docs/4.3/layout/grid/index.html#order-classes